### PR TITLE
Fix adventure border rendering on higher resolutions

### LINKED ISSUE #1286: Redundant embranchments of frames around status window
When game is running on any higher that 640x480 resolution, we can see two embranchments of the status window frame. 
UI frame

### PR DESCRIPTION
**Pull request by a1exsh**
Fix #1286

### CODE REVIEW

**Comment by a1exsh** [comment]
Good:
Screenshot from 2021-09-11 10-12-27

Evil:
Screenshot from 2021-09-11 10-12-59

File: src/fheroes2/gui/interface_border.cpp [hunk 1]
```diff
@@ -117,6 +117,17 @@ void Interface::GameBorderRedraw( const bool viewWorldMode )
     srcrt.height = icnadv.height() - srcrt.y;
     fheroes2::Blit( icnadv, srcrt.x, srcrt.y, display, dstpt.x, dstpt.y, srcrt.width, srcrt.height );
 
+    // fix extra border part on higher resolutions
+    if ( displayHeight > fheroes2::Display::DEFAULT_HEIGHT ) {
+        srcrt.x = 478;
+        srcrt.y = conf.ExtGameEvilInterface() ? 328 : 345;
```
Comment (ihhub) [inline src/fheroes2/gui/interface_border.cpp:123]: Please cache the value of `ExtGameEvilInterface()` call as it's not an inline function. You could do something like this:
```cpp
const bool isEvilInterface = conf.ExtGameEvilInterface();
```
Please declare such variable before line 53.

File: src/fheroes2/gui/interface_border.cpp [hunk 2]
```diff
@@ -137,6 +148,17 @@ void Interface::GameBorderRedraw( const bool viewWorldMode )
     srcrt.height = icnadv.height() - srcrt.y;
     fheroes2::Blit( icnadv, srcrt.x, srcrt.y, display, dstpt.x, dstpt.y, srcrt.width, srcrt.height );
 
+    // fix extra border part on higher resolutions
+    if ( displayHeight > fheroes2::Display::DEFAULT_HEIGHT ) {
+        srcrt.x = 624;
+        srcrt.y = conf.ExtGameEvilInterface() ? 328 : 345;
+        srcrt.width = 3;
+        srcrt.height = conf.ExtGameEvilInterface() ? 15 : 20;
+        // dstpt.x += 0;
```
Comment (ihhub) [inline src/fheroes2/gui/interface_border.cpp:157]: Please remove this commented line.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @a1exsh , I left 2 comments here. Would you mind to take a look?

**Comment by a1exsh** [comment]
@ihhub good points, fixed.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@a1exsh , thank you so much for this fix!

**Pull request merged**